### PR TITLE
fix: fixed axios vulnerabilities

### PR DESCRIPTION
**Pull request by OrrLumigo**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
   },
   "dependencies": {
     "agentkeepalive": "^4.1.4",
-    "axios": "0.21.1",
+    "axios": "^0.24.0",
```
Comment (efimk-lu) [inline package.json:25]: Why not upgrade to 0.21.2?

Comment (OrrLumigo) [reply package.json]: i ran "npm audit fix" that suppose to fix vulnerabilities and thats what it outputed but changed it now manually

File: src/httpSpansAgent.test.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ describe('HttpSpansAgent', () => {
 
     const requests = AxiosMocker.getAxiosMocker().history.post;
     expect(requests.length).toEqual(1);
-    expect(requests[0].data).toEqual(reqBody);
+    expect(JSON.parse(requests[0].data)).toEqual(reqBody);
```
Comment (efimk-lu) [inline src/httpSpansAgent.test.js:41]: @OrrLumigo how will affect the data arriving to our BE?

Comment (OrrLumigo) [reply src/httpSpansAgent.test.js:41]: this is just the Axios mocker its not the actual Axios lib

**Review by doriaviram — APPROVED** [review]

**Pull request merged**